api/bull-board/uaaClient.js

```diff
@@ -0,0 +1,146 @@
+const HttpClient = require('../utils/httpClient');
+const { uaa: uaaOptions } = require('../../config/passport');
```

Comment:
@amirbey this looks like the wrong config/passport or have all your changes not been pushed up?